frontend/src/v5/ui/components/dashboard/dashboardList/dashboardList.styles.ts

```diff
@@ -72,3 +73,13 @@ export const DashedButtonContainer = styled(DashedContainer).attrs(({ theme }) =
 	margin-top: 18px;
 	background-color: transparent;
 `;
+
+export const DashboardLink = styled(Link)`
+	&:link { text-decoration: none; }
+	&:visited {
+		text-decoration: none;
+		color:inherit;
+	}
+	&:hover { text-decoration: none; }
+	&:active { text-decoration: none; }
+`;
```

Comment:
I often happen to apply these CSS styles to `Link`s too, would it be worth it to create a `@control` component that already comes with these styles?

Reply:
Also, adding `color: inherit;` to `&:link` should solve the next mentioned problem